Node/taskqueues-backup-images/functions/index.js

```diff
@@ -25,6 +26,7 @@ const {getStorage} = require("firebase-admin/storage");
 const {GoogleAuth} = require("google-auth-library");
 const logger = functions.logger;
 const HttpsError = functions.https.HttpsError;
```

Comment:
Strictly for TQ:

const {onTaskDispatched} = require("firebase-functions/v2/tasks");
const {onRequest} = require("firebase-functions/v2/https");
const {getFunctions} = require("firebase-admin/functions");
const logger = functions.logger; (only do this the right way, like in the callable sample)